src/safe_apps/views.py

```diff
@@ -52,7 +65,13 @@ def get(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         return super().get(request, *args, **kwargs)
 
     def get_queryset(self) -> QuerySet[SafeApp]:
-        queryset = SafeApp.objects.filter(visible=True)
+        only_listed = parse_boolean_query_param(
+            self.request.query_params.get("onlyListed", True)
+        )
+        if not only_listed:
+            queryset = SafeApp.objects.all()
+        else:
+            queryset = SafeApp.objects.filter(visible=True)
```

Comment:
nitpick: I'd use the positive "form" / expression instead of negating it.

```python
if only_listed:
    queryset = SafeApp.objects.filter(visible=True)
else:
    queryset = SafeApp.objects.all()
```

Reply:
I agree, thanks!  Done in [501ce8f](https://github.com/safe-global/safe-config-service/pull/1111/commits/501ce8ffbbed691e5c1e3b48424860ba45489a29)